test/testdrive/bad_sql.td

```diff
@@ -0,0 +1,14 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Check that
+# https://github.com/MaterializeInc/materialize/issues/2880
+# is fixed. If not, this will hang.
+! asdfjkl;
+Parse error:
```

Comment:
I'm skeptical that this test does the correct thing! Testdrive does not use the simple query path. I think this needs to go in pgwire.rs.

Reply:
argh, you're right :/